internal/txtdiff/diff.go

```diff
@@ -0,0 +1,112 @@
+// Copyright 2018 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package txtdiff provides text-related test helpers.
+package txtdiff
+
+import (
+	"flag"
+	"io/ioutil"
+	"strings"
+	"testing"
+)
+
+var updateGolden = flag.Bool("update_golden", false, "update golden files")
+
+func Diff(t *testing.T, name, got, goldenFile string) {
```

Comment:
I know I'm late to the party, but a brief description of this function would be helpful.